webcompat/form.py

```diff
@@ -154,6 +159,22 @@ def normalize_url(url):
     return url
 
 
+def normalize_metadata(metadata_value):
+    """Normalize the metadata received from the form."""
+    # Removing closing comments.
+    if metadata_value is None:
+        return None
+    if '-->' in metadata_value:
+        metadata_value = metadata_value.replace('-->', '')
+        metadata_value = normalize_metadata(metadata_value)
```

Comment:
so if we found a `-->`, remove it and recurse checking the modified value. I guess `blue sky ---->>` from the test catches where you might have two closing quote strings (in a more clever way). Cool.

Reply:
exactly for for this test: `blue sky ---->>`

1. `blue sky ---->>` (first pass `'-->' in metadata_value`). True. replace
2. `blue sky -->` (second pass `'-->' in metadata_value`). True. replace
3. `blue sky ` (third pass `'-->' in metadata_value`) False. next test.

Reply:
if you want I can add another test. but note that in the case of two closing comments non-nested. They will be removed right away in one pass.

```python
>>> metadata_value = 'blue sky -->-->'
>>> metadata_value.replace('-->', '')
'blue sky '
```